pom.xml

```diff
@@ -293,9 +204,6 @@
 							<goal>integration-test</goal>
 						</goals>
 						<configuration>
-							<includes>
```

Comment:
I did not find any unit test or IT test code in this project.  Maybe we can get rid of the maven-failsafe-plugin